woo-viet.php

```diff
@@ -84,6 +84,9 @@ class WooViet {
 	 */
 	public function __construct() {
 		add_action( 'init', array( $this, 'init' ) );
+
+		// Enqueue scritps
+		add_action( 'wp_enqueue_scripts', array( $this, 'load_scripts' ) );
```

Comment:
If you load the scripts in the main file of the plugin, the user has no way to disable this. I want this feature attaching to the Provinces feature.